### ISSUE TITLE
Unhandled rejection TypeError: Arguments to path.resolve must be strings

### ISSUE THREAD

**Issue by gingerlime** [megawac/lodash-modularize#8]
running `lodash-modularize utils.js --list` produces this error (tried different file variations, all the same):

```
Unhandled rejection TypeError: Arguments to path.resolve must be strings
    at Object.exports.resolve (path.js:313:15)
    at exports.relative (path.js:371:18)
    at exports.default (/home/yoav/.nvm/v0.10.33/lib/node_modules/lodash-modularize/dist/parseForModules.js:102:118)
    at /home/yoav/.nvm/v0.10.33/lib/node_modules/lodash-modularize/dist/lodash-modularize.js:42:47
From previous event:
    at /home/yoav/.nvm/v0.10.33/lib/node_modules/lodash-modularize/dist/lodash-modularize.js:41:48
From previous event:
    at resolve (/home/yoav/.nvm/v0.10.33/lib/node_modules/lodash-modularize/dist/lodash-modularize.js:40:32)
    at /home/yoav/.nvm/v0.10.33/lib/node_modules/lodash-modularize/dist/lodash-modularize.js:51:12
    at process._tickCallback (node.js:419:13)
    at Function.Module.runMain (module.js:499:11)
From previous event:
    at modularize (/home/yoav/.nvm/v0.10.33/lib/node_modules/lodash-modularize/dist/lodash-modularize.js:50:107)
    at Object.<anonymous> (/home/yoav/.nvm/v0.10.33/lib/node_modules/lodash-modularize/bin/lodash-modularize:131:1)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:906:3
```

**Comment by megawac** [comment]
Hi, where is utils.js coming from? Is it available somewhere? Thanks.

**Comment by gingerlime** [comment]
It's just a local javascript file. The same error happens even if I pass a different file / empty js file / folder etc

**Comment by megawac** [comment]
Thanks, should be fixed in `v1.3.1`

**Comment by gingerlime** [comment]
Thanks! :+1:
